bokehjs/src/coffee/common/embed.coffee

```diff
@@ -8,6 +8,18 @@ HasProperties = require "./has_properties"
 {pull_session} = require "./client"
 {Promise} = require "es6-promise"
 
+_handle_notebook_comms = (msg) ->
+  logger.debug("handling notebook comms")
+  # @ is bound to the doc
+  @apply_json_patch_string(msg.content.data)
+
+_init_comms = (target, doc) ->
+  logger.info("Initializing Jupyter comms for target #{target} and doc #{doc}")
+  comm_manager = Jupyter.notebook.kernel.comm_manager
```

Comment:
spoke too soon, that's not working reliably. I imagine because `require` is asynchronous, so sometimes the python tries to connect before the JS comms is done or even starts to get registered. (se below) So I'm just going to put it back the way it was. Please implore them not to remove important things from `Jupyter`. It's of no benefit that I can see to do do, and I have no idea how any one would ever be able to reliably make a python connection work if they can't at least assume the JS target has been already been registered. 

<img width="473" alt="screen shot 2015-12-21 at 3 26 38 pm" src="https://cloud.githubusercontent.com/assets/1078448/11941739/68420d12-a7f7-11e5-9a18-392a73045f7b.png">